client/components/NavBar.js

```diff
@@ -2,21 +2,29 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 
+import Notification from './Notification';
+
 export class NavBar extends React.Component {
   constructor(props){
     super(props);
 
     this.logOut = this.logOut.bind(this);
   }
-
+  /**
+   * [logOut description]
+   * @method logOut
+   * @return {[type]} [description]
+   */
   logOut() {
     window.localStorage.removeItem('token');
     window.location.href = '/';
+    alert(1234);
   }
 
   render() {
     const status = this.props.logged_in_status;

```

Comment:
You might want to use camelCase for variable naming eg loggedInStatus as opposed logged_in_status